jws/jws.go

```diff
@@ -424,40 +431,64 @@ func ParseString(s string) (*Message, error) {
 	return Parse(strings.NewReader(s))
 }
 
-func parseJSON(src io.Reader) (result *Message, err error) {
+type fullMessageProxy struct {
+	// encoded signature fields
+	Signature json.RawMessage `json:"signature"`
+	Headers   json.RawMessage `json:"header"` // あれ、"s"いらないんだっけ
```

Comment:
Not sure what that comment means (katakana?)

Reply:
I was wondering if this was `header` or `headers` :)

Reply:
It's `header' for JSON serialization format :+1: